spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+feature  'i want to be able to sign up' do
+  scenario "user clicks on the submit button" do
+    visit new_user_registration_path
+    click_button 'Sign up'
+  end
```

Comment:
There should be an expect statement here. E.g. a flash message showing the success and an expect statement for the current pathname